src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,356 @@
+package main.java.duke;
+
+import main.java.duke.exception.EmptyTodoException;
+import main.java.duke.exception.InvalidCommandException;
+import main.java.duke.task.Deadline;
+import main.java.duke.task.Event;
+import main.java.duke.task.Task;
+import main.java.duke.task.TaskType;
+import main.java.duke.task.ToDo;
+
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Duke {
+    public static final int MAX_LIST_SIZE = 100;
+    private static ArrayList<Task> tasks = new ArrayList<>(MAX_LIST_SIZE);//user's to-do-list (max 100 items)
+    public static final int MAX_TASK_LENGTH = 4;
+    public static final String filepath = "./data/duke.txt";//file path from root file
+
+    //get list size
+    public static int getListSize() {
+        return tasks.size();
+    }
+
+    //retrieve past saved data
+    private static void handlePastData() {
+        File pastFile = new File(filepath); // create a File for the given file path
+        String wholeDoc = readFile(pastFile);
+        //when file is empty or not created
+        if (wholeDoc == null) {
+            return;//skip parsing data
+        }
+        //convert file entry to tasks
+        parseFile(wholeDoc);
+    }
+
+    //read file content
+    private static String readFile(File pastFile) {
+        String wholeDoc = null;
+        try {
+            //read previous data
+            Scanner s = new Scanner(pastFile);
+            while (s.hasNext()) {
+                wholeDoc = s.nextLine();
+            }
+        } catch (FileNotFoundException e) {
+            printFileNotFound();
+            return null;
+        }
+        return wholeDoc;
+    }
+
+    //display file not found error message
+    private static void printFileNotFound() {
+        System.out.println(" Are you first time user?\n" +
+                " I am unable to find your past data\n" +
+                "____________________________________________________________");
+    }
```

Comment:
Consider moving all your print functionalities to a separate class dedicated for printing out messages